ntbs-service/Models/ViewModels/TransferViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using ExpressiveAnnotations.Attributes;
+using Microsoft.AspNetCore.Mvc;
+using ntbs_service.Models.Validations;
+
+namespace ntbs_service.Models.ViewModels;
+
+public class TransferViewModel
+{
+    [BindProperty]
+    [Display(Name = "Transfer date")]
+    [Required(ErrorMessage = ValidationMessages.RequiredEnter)]
+    [ValidDateRange(ValidDates.EarliestBirthDate)]
```

Comment:
Since this date has nothing to do with birth here, perhaps we should rename it to "EarliestAllowedDate"?